assets/scripts/script.js

```diff
@@ -1 +1,44 @@
-/* jshint esversion: 11 */
\ No newline at end of file
+// window.onload = function() {
+
+//   if (localStorage) {
+
+ 
+//     document.getElementById('settings-form').addEventListener('submit', function() {
+  
+//       let difficultyLevel = document.getElementById('difficulty-level').value
```

Comment:
`document.querySelector('input[name="inlineRadioOptions"]:checked').value;`  look for checked in the radio group

Reply:
Thanks i'll try that

Reply:
Working now on my end anyway